subgraph/package.json

```diff
@@ -30,6 +30,7 @@
     "eslint": "8.31.0",
     "fs-extra": "11.1.0",
     "handlebars": "4.7.7",
+    "keccak256": "1.0.6",
```

Comment:
Let's use ethers here, as it is already a dependency in the repo and has the right functions for keccak256